sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -4,11 +4,13 @@
 
 ### Features Added
 
-### Breaking Changes
+- Keyvault 7.3 support added for Certificates.
 
-### Bugs Fixed
+## 4.0.0 (2022-06-07)
 
-### Other Changes
+### Breaking Changes
+
+- Renamed `keyvault_certificates.hpp` to `certificates.hpp`.
 
```

Comment:
Looks like a bad merge, 4.0.0. is duplicated here.

Reply:
done